pages/1.10/release-notes/1.10.4/index.md

```diff
@@ -23,10 +23,10 @@ These are the release notes for DC/OS 1.10.4.
 
 # Notable Changes in DC/OS 1.10.4
 
-- Mesos 1.4.1 is integrated with DC/OS 1.10.4.
-- Marathon 1.5.5 is integrated with DCOS 1.10.4.
-- DC/OS Java Developer Kit upgraded to the latest version: JDK 8u152.
-- DC/OS is compatible with the newest Docker version: 17.05.0.
+- Updated to Mesos 1.4.1 ([changelog](https://github.com/apache/mesos/blob/1.4.x/CHANGELOG)).
```

Comment:
Mesos 1.4.1 was introduced in DC/OS 1.10.5, not before, right? cc @orsenthil 
see https://github.com/dcos/dcos/pull/2382